addons/account_asset/models/account_invoice.py

```diff
@@ -108,3 +108,9 @@ def _set_additional_fields(self, invoice):
             elif invoice.type == 'in_invoice':
                 self.asset_category_id = self.product_id.product_tmpl_id.asset_category_id.id
         super(AccountInvoiceLine, self)._set_additional_fields(invoice)
+
+    @api.v8
```

Comment:
Not necessary, see https://github.com/odoo/odoo/blob/saas-14/addons/stock_account/models/account_invoice.py#L99